contrib/miniupnpd/template.py

```diff
@@ -0,0 +1,38 @@
+pkgname = "miniupnpd"
+pkgver = "2.3.6"
+pkgrel = 0
+build_wrksrc = pkgname
+build_style = "configure"
+configure_args = [
+    "--firewall=nftables",
+    "--ipv6",
+    "--portinuse",
+]
+make_cmd = "gmake"
+hostmakedepends = [
+    "git",
```

Comment:
as far as i can tell git is unused here

Reply:
Yeah, I've added it while doing development and forgot to remove

Reply:
same for procps i think

Reply:
It uses `sysctl` binary from procps

Reply:
in a bad way- it uses the build server configuration to configure itself for --v6sockets-v6only

Reply:
Do you think it would be better to just patch to remove that part of the code?

Reply:
i'm not sure, i think the daemon itself is hardcoded to only work with one configuration so the other would be broken at runtime

but most people are probably not running v6only so it's better to patch it out to guarantee that it doesn't enable that i guess